src/github.com/matrix-org/go-neb/database/schema.go

```diff
@@ -262,6 +262,35 @@ func selectRealmTxn(txn *sql.Tx, realmID string) (types.AuthRealm, error) {
 	return realm, nil
 }
 
+const selectRealmsByTypeSQL = `
+SELECT realm_id, realm_json FROM auth_realms WHERE realm_type = $1
```

Comment:
Would it make sense to throw an order-by into this query so that the results come out in a consistent order?